src/tests/__snapshots__/Details.spec.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Details page renders 1`] = `
+<div
+  className="container"
+/>
+`;
```

Comment:
- [ ] Your Application should have unit tests for several components of the app and some pure functions in Redux code. Please add this ✍🏻 

- [ ] The Application should also have integration tests using React Testing Library. You can perform a mocking data fetching and Redux Store connection. Please add this too ✍🏻